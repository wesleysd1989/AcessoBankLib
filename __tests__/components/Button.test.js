import React from 'react';
import renderer from 'react-test-renderer';

import {Button} from '../../lib';

describe('Button', () => {
  test('Deve ocorrer um match entre o componente Button e seu snapshot quando são passados valores padrões.', () => {
    const tree = renderer
      .create(<Button onPress={() => null}>Test</Button>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
