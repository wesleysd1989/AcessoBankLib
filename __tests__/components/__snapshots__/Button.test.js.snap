// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Deve ocorrer um match entre o componente Button e seu snapshot quando são passados valores padrões. 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#00A074",
        "justifyContent": "center",
        "minHeight": 60,
      },
    ]
  }
>
  <Text
    disabled={false}
    numberOfLines={1}
    style={
      Array [
        Object {
          "color": "#FFF",
          "fontFamily": "DINRoundPro-Medium",
          "fontSize": 18,
          "letterSpacing": -0.4,
          "textAlign": "center",
        },
      ]
    }
    type="primary"
  >
    Test
  </Text>
</View>
`;
